### ISSUE TITLE
Advanced setting Clear Blockchain

### ISSUE THREAD

**Issue by Pem199** [project-ecc/sapphire#84]
When clicking on the clear the blockchain button it shows a daemon stopped succesful toast. Then it tries to clear the chain and it fails to delete the chain. Then it tries to restart the daemon but fails.

It seems that this is caused by a problem that was also reported in issue #83 . The daemon stopped signal is fired to early. So sapphire thinks the daemon is stopped even though its not yet. The process fails, tries to restart the daemon, but since it is still shutting down it fails there aswell

**Comment by gl3nda85** [comment]
can you double check that this commit fixes the problem 5300e6103b6e2256e2180d84d26188b1c30acd81

**Comment by Pem199** [comment]
No, after the message the daemon is shutdown, which still seems to be to early, it takes a few seconds and then it gives a red toast saying that it couldnt delete the chain, and then it ask to start the daemon again. Most files of the blocks folder were removed but some werent. This causes the daemon to not be able to start.

Looking at the console it gives the following message when i get the red toast:
Error: EPERM: operation not permitted, lstat 'C:\Users\pim_h\Appdata\roaming\eccoin\blocks\index\LOCK' Advanced.js:147

**Comment by gl3nda85** [comment]
yeah this is due the the daemon not being stopped and having a PID lock

**Comment by Dolaned** [comment]
https://github.com/project-ecc/sapphire/commit/790d190905b42a74360e6245517894a1e63ef279

added a delay please check this @Pem199
